src/sections/Projects/Project-grid/index.js

```diff
@@ -70,7 +80,10 @@ const ProjectPage = () => {
             </Link>
           </div>
 
-          <ContactFormModal callout_text="Contact Us" form_header="Get In Touch" />
+          <CommunityCallCard
+            callout_text="Join community"
+            form_header="Get In Touch"
```

Comment:
This component doesn't require these props. It can be removed and it can be called only like this `CommunityCallCard />`